src/main/java/Deadline.java

```diff
@@ -0,0 +1,47 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    LocalDate time;
```

Comment:
Should this variable be private (with a separate getter method to retrieve this field in the class)? 🤔